orttraining/orttraining/test/python/orttraining_test_ortmodule_bert_classifier.py

```diff
@@ -431,5 +441,43 @@ def main():
     print("  Accumulated training took:                {:.4f}s".format(total_training_time))
     print("  Accumulated validation took:              {:.4f}s".format(total_test_time))
 
+    if args.log_perf_metrics:
+        perf_metrics = {}
+        perf_metrics['Model'] = 'orttraining_test_ortmodule_bert_classifier'
+        perf_metrics['BatchId'] = 'na'
```

Comment:
assuming you'll use perf_metrics_start_time to publish this right?
this will be used for BI charts